### PR TITLE
fix(Global Tagging): add 'update' option to attach_tag operation

### PR DESCRIPTION
**Pull request by fabrizio-leoni**
## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by fabrizio-leoni** [comment]
image

**Comment by fabrizio-leoni** [comment]
image

File: test/unit/global-search.v2.test.js [hunk 1]
```diff
@@ -17,7 +17,7 @@
 // need to import the whole package to mock getAuthenticatorFromEnvironment
 const sdkCorePackage = require('ibm-cloud-sdk-core');
 
-const { NoAuthAuthenticator } = sdkCorePackage;
```
Comment (padamstx) [inline test/unit/global-search.v2.test.js:20]: Please install the latest version of the SDK generator and re-gen your code.  There should be no changes to this unit test file.

Note: recent changes were made to the node core library and the SDK generator to separate out some unit test utilities into their own npm package (`@ibm-cloud/sdk-test-utilities`).  By installing the latest generator version, you should get the correct code generated in the unit tests.

File: test/unit/global-tagging.v1.test.js [hunk 2]
```diff
@@ -17,7 +17,7 @@
 // need to import the whole package to mock getAuthenticatorFromEnvironment
 const sdkCorePackage = require('ibm-cloud-sdk-core');
 
-const { NoAuthAuthenticator } = sdkCorePackage;
+const { NoAuthAuthenticator, unitTestUtils } = sdkCorePackage;
```
Comment (padamstx) [inline test/unit/global-tagging.v1.test.js:20]: Need to re-gen with the most recent SDK generator version and these import-related changes should go away.

**Review by padamstx — CHANGES_REQUESTED** [review]

File: test/unit/global-search.v2.test.js (hunk 1 above)
Comment (padamstx) [reply test/unit/global-search.v2.test.js:20]: Please disregard my previous comment.  I have updated the two unit tests manually

**Review by padamstx — APPROVED** [review]
After my manual changes to the unit tests, everything looks good.

File: test/unit/global-search.v2.test.js (hunk 1 above)
Comment (fabrizio-leoni) [reply test/unit/global-search.v2.test.js:20]: Resolved by @padamstx

File: test/unit/global-tagging.v1.test.js (hunk 2 above)
Comment (fabrizio-leoni) [reply test/unit/global-tagging.v1.test.js]: Resolved by @padamstx

**Pull request merged**